src/Core/Request.php

```diff
@@ -4,75 +4,11 @@
 
 class Request {
 
-    /**
-     * @var array
-     */
-    public $parameters = [];
-
-
-    /**
-     * Request parameters
-     * @var array
-     */
-    public $params = [];
-
-
-    /**
-     * Request clearn uri
-     * @return mixed|string
-     */
-    public static function uri() {
-        $uri = trim($_SERVER['REQUEST_URI'], '/');
-
-        if(!empty($_SERVER['REQUEST_QUERY'])){
-            $uri= str_replace( $_SERVER['REQUEST_QUERY'], '', $uri );
-        }
-
-        if(!empty($_SERVER['QUERY_STRING'])) {
-            $uri= str_replace( $_SERVER['QUERY_STRING'], '', $uri );
-        }
-
-        $uri = trim($uri, '?');
-
-
-        return $uri;
-
-    }
-
-
-    /**
-     * Return request method type
-     * @return mixed
-     */
-    public static function methodType() {
+    public function methodType() : string {
         return $_SERVER['REQUEST_METHOD'];
     }
 
-
-    /**
-     * Return Request parameters
-     *
-     * @return array
-     */
-    public function parameters() {
-        if(!empty($_SERVER['QUERY_STRING'])){
-            $url = $_SERVER['QUERY_STRING'];
-            $this->parameters = explode('&', $url);
-            array_map(function ($arg){
-                $temp = explode('=', $arg);
-                if(!isset($this->params[$temp[0]])) {
-                    return;
-                }else {
-                    $this->params[$temp[0]] = $temp[1];
-                }
-
-            }, $this->parameters);
-            return $this->params;
-        }
-
-
+    public function getSlug() : string {
+        return $uri = $_SERVER['PATH_INFO'] ?? '/';
```

Comment:
I wanted to return the url without any query strings like if i had 
`/test?p=1` 
`$_SERVER['PATH_INFO']` should return `/test` but if i passed nothing as hitting the root of the project `localhost:8000/` the `$_SERVER['PATH_INFO']` will be empty so this will return `/`

Reply:
The intention is clear, I'm talking specifically about `$uri =` part. Why do you need this local variable?

Reply:
Ok sorry, You are right